rust/parquet/Cargo.toml

```diff
@@ -49,7 +49,6 @@ brotli = "3.3"
 flate2 = "1.0"
 lz4 = "1.23"
 zstd = "0.5"
-arrow = { path = "../arrow", version = "2.0.0-SNAPSHOT" }
```

Comment:
hmm, does this mean we're removing arrow dependency from tests as well?

Reply:
yes, because it is already a dependency in `arrow.rs`, using `super::*;` export in test modules will make it use the actual dependency exports of the arrow (which has been done already seems like). Tests had passed locally with no change and it passed here. I see it as good improvement tbh :)